ext/symengine/symengine.c

```diff
@@ -113,6 +113,10 @@ void Init_symengine() {
     c_dirichlet_eta = rb_define_class_under(m_symengine, "Dirichlet_eta", c_function);
     c_zeta = rb_define_class_under(m_symengine, "Zeta", c_function);
     c_gamma = rb_define_class_under(m_symengine, "Gamma", c_function);
+    c_one_arg_function = rb_define_class_under(m_symengine, "OneArgFunction", c_function);
+
+    //Abs Class
+    c_abs = rb_define_class_under(m_symengine, "Abs", c_function);
```

Comment:
I was thinking it's better to keep the same class hierarchy as in C++, maybe it will be useful in the future. Currently there's absolutely no practical use though. What do you think? I can get rid of it if you want.